src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define CREATE_LOCAL_LOGGER(logger) \
+  src::logger_mt logger
+
+#define CREATE_GLOBAL_LOGGER(logger) \
+namespace { \
+  GET_LOCAL_LOGGER(); \
```

Comment:
@AGaliuzov, seems you're trying to use incorrect macro. You have `CREATE_LOCAL_LOGGER`, not `GET_LOCAL_LOGGER`

Have you tested how logger works?

As far as I understand boost is not an hierarchical logger. so it has no names. But there is some "scope" feature. I think we need something to mark the origin of the log lines.